src/components/InputLabel/InputLabel.tsx

```diff
@@ -27,6 +28,7 @@ const TOOLTIP_PADDING = 15;
 export const InputLabel: FC<InputLabelProps> = ({
     children,
     htmlFor,
+    clickable = false,
```

Comment:
Can we base ourselves on `htmlFor` instead of creating a new one?
As when `htmlFor` is set, we are able to click